installing/install_config/customizations.adoc

```diff
@@ -26,49 +26,68 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configuration resource |Description
 
 |APIServer
-|Configure the Kubernetes API server.
+|You can
+xref:../../authentication/certificates/api-server.adoc#api-server-certificates[add certificates]
+for the API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
-|Configure build settings.
+|You can configure your
+xref:../../builds/build-configuration.adoc#build-configuration[build] settings.
 
 |ClusterVersion
-|Configure the desired version of the cluster and how updates are handled.
+|In {product-title} {product-version}, you must not customize the ClusterVersion
+resource for production clusters. Instead, follow the process to
+xref:../../updating/updating-cluster.adoc#updating-cluster[update a cluster].
 
 |Console
 |Configure the web console.
```

Comment:
I am adding a link to the new console config topic on this PR: https://github.com/openshift/openshift-docs/pull/15097